packages/react-server-dom-esm/src/ReactFlightESMNodeLoader.js

```diff
@@ -329,9 +329,9 @@ async function transformClientModule(
       newSrc +=
         'throw new Error(' +
         JSON.stringify(
-          `Attempted to call the default export of ${url} from the server` +
+          `Attempted to call the default export of ${url} from the server ` +
```

Comment:
I think the original one is better, we don't have to add the space here

Reply:
No, you do need the space. Without it, this renders as `...from the serverbut it's on the client... to props of aClient Component.`

Reply:
> No, you do need the space. Without it, this renders as `...from the serverbut it's on the client... to props of aClient Component.`

Sorry that I didn't notice, you are correct!